### ISSUE TITLE
Query on analysing sources not in 4FGL catalog

### ISSUE THREAD

**Issue by sushmitaagarwal** [fermiPy/fermipy#549]
## Data Analysis Question

I want to look for high-energy gamma-ray photons from a source that is not in the 4FGL catalog. As the source is not in the catalog, I cannot provide a target name, but I do have its RA and DEC coordinates.

My goal is to generate TS (Test Statistic) maps for this source and determine if there is a significant gamma-ray emission. However, Fermipy requires a source name to free parameters and perform the likelihood analysis, which poses a challenge for sources not in the catalog.

However, if I don't have a name in the 4FGL catalog, I cannot free the parameters since I need a source name for that. I would like to know if there is a possible workaround to analyze sources not listed in the 4FGL catalog using Fermipy. 

Kindly let me know if it is possible using fermipy. Any insights or solutions would be greatly appreciated.

**Comment by lylol** [comment]
Hi @sushmitaagarwal, I guess the add_source function maybe of use. Link: https://fermipy.readthedocs.io/en/latest/model.html.

**Comment by omodei** [comment]
Yes, @lylol is correct,. @sushmitaagarwal can this issue be closed?

**Comment by sushmitaagarwal** [comment]
Yess. Thanks for help.
